src/source/Listener/GUDButtonListener.cpp

```diff
@@ -16,7 +16,11 @@ void GUDButtonListener::onButton1MouseClick(BaseButtonInstance* button)
 		}
 	if(cont)
 	{
-		AudioPlayer::playSound(dingSound);
+		
+		if(button->disabled == false){
+			AudioPlayer::playSound(dingSound);
```

Comment:
This is old code, but updating it to use the new system is a good idea. Though, these UI sounds might play through the win32 playsound function in the future, rather than the SDL or future Fmod sound systems

Reply:
I did implement FMOD, but as others have said an I agree, FMOD is most likely not accurate. I do believe Win32 playsound is the way to go.